docs/reference/glossary.md

```diff
@@ -100,19 +122,20 @@ lightweight Docker development environment designed specifically for the Mac. A
 native Mac application, Docker for Mac uses the macOS Hypervisor framework,
 networking, and filesystem. It's the best solution if you want to build, debug,
 test, package, and ship Dockerized applications on a Mac. Docker for Mac
-supersedes [Docker Toolbox](#toolbox) as state-of-the-art Docker for macOS.
```

Comment:
Incomplete sentence; Looks like something went wrong here with the rebase, or the trailing `Docker for Mac` must be removed